client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -259,6 +259,7 @@ export default function getThankYouPageUrl( {
 					siteId: jetpackTemporarySiteId && parseInt( jetpackTemporarySiteId ),
 					fromSiteSlug,
 					productSlug,
+					redirect_to: redirectTo,
```

Comment:
Is it possible to add a test to cover this case?

https://github.com/Automattic/wp-calypso/blob/2cf3125b1e2c79c390b9812968f4c1009c0c6326/client/my-sites/checkout/get-thank-you-page-url/index.ts#L2-L7

Reply:
Will do. Thanks @sirbrillig 👍

Reply:
Addressed in 55a8545012